Dockerfile

```diff
@@ -16,6 +19,8 @@ RUN git submodule init && git submodule update --remote --recursive --merge
 
 RUN go build
 
+RUN cd web && npm install --global gulp-cli && npm install && gulp
```

Comment:
i think there may be some benefit to splitting the lines in that it might cache the steps separately, but w/e for now